src/py/flwr/common/recordset_test.py

```diff
@@ -362,4 +358,4 @@ def test_set_configs_to_configsrecord_with_incorrect_types(
     )
 
     with pytest.raises(TypeError):
-        m_record.set_configs(my_metrics)  # type: ignore
+        m_record.update(my_metrics)
```

Comment:
It was like that before, but shouldn't it be `c_record` instead of `m_record`?
```suggestion
        c_record.update(my_metrics)
```